source/includes/_accounts.md

```diff
@@ -303,24 +299,21 @@ This feature is not accessible to Apollo users on free plans.
 > Sample request:
 
 ```shell
-curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" "https://api.apollo.io/v1/account_stages?api_key=YOUR_API_KEY_HERE"
+curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" -H "X-Api-Key: YOUR API KEY HERE" "https://api.apollo.io/v1/account_stages"
 ```
 
 ```python
 import requests
 
 url = "https://api.apollo.io/v1/account_stages"
 
-querystring = {
-  "api_key": "YOUR API KEY HERE"
-}
-
 headers = {
     'Cache-Control': 'no-cache',
-    'Content-Type': 'application/json'
+    'Content-Type': 'application/json',
+    'X-Api-Key': 'YOUR API KEY HERE'
 }
 
-response = requests.request("GET", url, headers=headers, params=querystring)
```

Comment:
The changes look good to me, please check this CURL request out once if this works without params being supplied.

Reply:
Yes I have verified it already by executing the python code in my local environment. It is working absolutely fine.